espnet2/tasks/spk.py

```diff
@@ -186,6 +200,10 @@ def add_task_arguments(cls, parser: argparse.ArgumentParser):
             help="Directory of spk2utt file to be used in label mapping",
         )
 
+        group.add_argument(
+            "--spk_num", type=int, default=None, help="spk number in training"
```

Comment:
Can you explain this option?
Do you mean that this is the number of speakers?
I thought that the number of speakers is computed by the occurrence of unique speaker IDs in the training data automatically.

Reply:
yes this is the number of speakers. We set this value automatically in `spk.sh` (https://github.com/Jungjee/espnet/blob/c77b6279f69b88c2705bd1bd6b5d7a1d223e7c3c/egs2/TEMPLATE/spk1/spk.sh#L440)

Reply:
Oh, I see. Then,
```suggestion
            "--spk_num", type=int, default=None, help="specify the number of speakers during training"
```